keel-api/keel-api.gradle.kts

```diff
@@ -10,6 +10,7 @@ plugins {
  * dependencies along for the ride -- ideally nothing at all.
  */
 dependencies {
+  api("com.netflix.spinnaker.kork:kork-core")
```

Comment:
Is this compatible with the pf4j plugin stuff? My understanding is we should not have any dependencies in here.

Reply:
Good question. I'll have to defer to @robzienert on that... I added this so I could use `SystemException` instead of `Throwable` in the handler interface.

Reply:
`kork-core` is definitely a no-no module. You can use `kork-exceptions`, however.

Reply:
Thanks! Good catch, @robfletcher!